NETWORK/NetworkFadeInEntity.md

```diff
@@ -5,20 +5,14 @@ ns: NETWORK
 
 ```c
 // 0x1F4ED342ACEFE62D 0x9B9FCD02
-void NETWORK_FADE_IN_ENTITY(Entity entity, BOOL state);
```

Comment:
If I recall correctly, you can't add parameters like this. There are some compatibility constraints with the ABI, hence why there's a fenced comment down below reading `NativeDB Added Parameter 3: BOOL slow`.